chromium_src/chrome/common/importer/firefox_importer_utils.cc

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// Clang format wants to place this header after the .cc file for some reason.
+// clang-format off
+#include "brave/grit/brave_generated_resources.h"
+
+#include "src/chrome/common/importer/firefox_importer_utils.cc"
+// clang-format on
```

Comment:
instead of disabling the clang-format entirely, please add a comment before `#include "src/chrome/common/importer/firefox_importer_utils.cc"` line to preserve the includes order. Something like:

```
#include "brave/grit/brave_generated_resources.h"

// A comment to keep src/**/.cc file after headers.
#include "src/chrome/common/importer/firefox_importer_utils.cc"
```

Reply:
Fixed in https://github.com/brave/brave-core/pull/26537/commits/a6b65dfc1cb66d13a4909acd7d9707ba81305129